prow/jobs/jinja/periodics/docs_release.jinja2

```diff
@@ -0,0 +1,32 @@
+name: docs-release
+  decorate: true
+  interval: 6h
```

Comment:
Shouldn't this be 24h?

Reply:
Good catch! thank you sir! :)

Reply:
I believe there is 24h, in another place as well.

Reply:
Yes we have other jobs that are scheduled to run every 6hours (label_sync, github issues stale/rotten plugins)

Reply:
Thanks for explaining, just wanted to confirm this is now 💯